llvm/lib/MC/MCGOFFStreamer.cpp

```diff
@@ -37,6 +117,15 @@ void MCGOFFStreamer::changeSection(MCSection *Section, uint32_t Subsection) {
   }
 }
 
+void MCGOFFStreamer::emitLabel(MCSymbol *Symbol, SMLoc Loc) {
+  MCObjectStreamer::emitLabel(Symbol, Loc);
+}
```

Comment:
If you just don't override this function, it should have the same effect, right?

Reply:
Yeah, leftover from some early code. Removed.